client/src/components/GustMenu/GustMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Button, Typography } from '@mui/material';
+import { Link, useHistory } from 'react-router-dom';
+import useStyles from './useStyles';
+
+interface Props {
+  loggedIn: boolean;
+}
+
+const GuestMenu = ({}: Props): JSX.Element => {
+  const classes = useStyles();
+  const history = useHistory();
+  return (
+    <Box justifyContent={'center'}>
+      <Link to={'/'} className={classes.linkBtn}>
+        <Typography display={'inline'} fontWeight={800} fontSize={11}>
+          BECOME A SITTER
+        </Typography>
```

Comment:
MUI v5 has a `sx` property that is generally preferred now. Example below. 

```js
<Typography sx={{ display: 'inline', fontWeight: 800, fontSize: 11 }}>
    BECOME A SITTER
</Typography>
```

Reply:
I think I make little mistakes when branching. I am working on the backend also When the backend is reviewing, I created branch from the backend branch.